sailon/novelty_interface.py

```diff
@@ -0,0 +1,31 @@
+import abc
+from typing import Tuple
+
+#from smqtk_core import Plugfigurable
+from smqtk_core import Configurable, Pluggable
+
+
+class ONDAdapter(Configurable, Pluggable, abc.ABC):
+    @abc.abstractmethod
+    def get_config(self):
+        raise NotImplementedError
```

Comment:
I don't think it is necessary to raise the `NotImplementedError` these methods can't be called due to the decorator.

Reply:
Good call, raising `NotImplementedError` is redundant. New commits replace this with a docstring in each abstract method body.